src/Microsoft.Azure.WebJobs.Host/Hosting/WebJobsHostBuilderExtensions.cs

```diff
@@ -160,6 +161,7 @@ public static IHostBuilder ConfigureWebJobs(this IHostBuilder builder, Action<Ho
                     services.Configure(configureScaleOptions);
                 }
 
+                services.AddFeatureManagement();
```

Comment:
Suggest we use the feature APIs built into .NET core https://learn.microsoft.com/en-us/azure/azure-app-configuration/use-feature-flags-dotnet-core?tabs=core5x. It's a clean model that composes with the configuration system.